docs/user_guide/configuration.rst

```diff
@@ -160,3 +204,22 @@ When setting logger tags for ``idaes.logging`` loggers they are compared against
 a list of valid tags.  This is done to guard against spelling errors. If the default
 set of defined tags is not sufficient tags can be added here, or later through
 functions in the ``idaes.logging`` module.
+
+Changes do not require ``idaes.reconfig()``.
+
+ipopt-idaes
+~~~~~~~~~~~
+This is a config block that provides default configuration for the ``ipopt-idaes``
+which is just a wrapper for the Pyomo ``ipopt`` solver class that allows the
+default solver options to be configured in the general idaes ConfigBlock.
```

Comment:
I think you should split this into two sentences: This is a config block that provides the default configuration for the ``ipopt-idaes`` solver. ``ipopt-idaes`` is a wrapper for the Pyomo ``ipopt`` solver class that allows default solver options to be configured in the general idaes ConfigBlock.